Regression/Checksum/benchmarks_json/hard_edged_quadrupoles_moving.json

```diff
@@ -1,22 +1,22 @@
 {
-  "electron": {
-    "particle_momentum_x": 2.0819392998019253e-25,
-    "particle_momentum_y": 2.3316091155621773e-33,
-    "particle_momentum_z": 2.730924530757351e-23,
-    "particle_position_x": 0.034923287741880055,
-    "particle_position_y": 9.300626840674331e-11,
-    "particle_position_z": 1.491521766472402,
-    "particle_weight": 1.0
-  },
   "lev=0": {
     "Bx": 0.0,
     "By": 0.0,
     "Bz": 0.0,
-    "Ex": 6.028256519009052e-05,
-    "Ey": 6.384796595673982e-05,
-    "Ez": 7.88045921306518e-05,
-    "jx": 0.0,
-    "jy": 0.0,
-    "jz": 0.0
+    "Ex": 6.0282565190090465e-05,
+    "Ey": 6.38479659567398e-05,
+    "Ez": 7.880459213065183e-05,
+    "jx": 1.1659465170956616e-09,
+    "jy": 2.6115239381823616e-17,
+    "jz": 1.5293971084510288e-07
+  },
+  "electron": {
+    "particle_momentum_x": 2.0819392998019267e-25,
```

Comment:
Same question here - do you know why the particle quantities changed? (though only in the last two digits)